firmware/hw_layer/stepper.cpp

```diff
@@ -152,15 +155,19 @@ int StepperMotor::getTargetPosition() const {
 	return m_targetPosition;
 }
 
-void StepperMotor::setTargetPosition(int targetPosition) {
+void StepperMotor::setTargetPosition(float targetPosition) {
```

Comment:
Would be be better to use "percent_t" here?

Reply:
name is clarified